src/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

```diff
@@ -424,6 +424,7 @@ public ServiceType ServiceType
 
         private static bool CheckMachineName(string value)
         {
+            // string.Contains(char) is .NetCore2.1+ specific
             return !string.IsNullOrWhiteSpace(value) && value.IndexOf('\\') == -1;
```

Comment:
Nit, I guess this could be `!string.IsNullOrEmpty` since the whitespace case would fail the indexof as well.

Reply:
Not my code, but happy to change it

Reply:
Eh, probably not worth it